src/redux/slices/missionsSlice.js

```diff
@@ -4,7 +4,14 @@ import fetchMissions from '../../services/missionsService';
 const initialState = {
   isLoading: false,
   isFaild: false,
-  items: [{}],
+  items: [
+    {
+      mission_id: 2,
+      mission_name: 'Telstar',
+      reserved: true,
+      description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam, consequatur ut. Distinctio nostrum voluptatibus recusandae eum, velit dolores mollitia nam aliquid consectetur enim corporis perspiciatis repudiandae animi, sed hic tenetur!',
+    },
+  ],
```

Comment:
Do you think the items of the initial state should have some data?

Reply:
I added it so that the MissionView snapshot test would not be an empty snapshot, in production the initial state is overwritten so the default data doesn't show on the UI. Do you have any suggestions on how we can handle this?

Reply:
Ok, in that case, let me re-do the snapshot for the `MissionView` component